src/image.rs

```diff
@@ -86,6 +86,7 @@ impl From<io::Error> for ImageError {
 pub type ImageResult<T> = Result<T, ImageError>;
 
 /// Result of a decoding process
+#[derive(Debug)]
```

Comment:
My impression is that structs should generally implement Debug to make everyone's life easier. I used it while developing and left it in intentionally but it's not strictly needed.